### PR TITLE
Feature/cell info in transaction

### PR DESCRIPTION
**Pull request by Daryl-L**
https://github.com/Magickbase/ckb-explorer-public-issues/issues/657

### CODE REVIEW

File: src/pages/Address/RgbppAssets.tsx [hunk 1]
```diff
@@ -179,6 +179,9 @@ const getCellDetails = (cell: LiveCell, t: TFunction) => {
     ...cell,
     id: Number(cell.cellId),
     isGenesisOutput: false,
+    generatedTxHash: cell.txHash,
+    cellIndex: cell.cellIndex.toString(16),
```
Comment (Keith-CY) [inline src/pages/Address/RgbppAssets.tsx:183]: `0x` is missing after `toString(16)`

File: src/pages/Transaction/TransactionCell/styles.module.scss [hunk 2]
```diff
@@ -10,3 +12,303 @@
...
+    color: #000;
+  }
+
+  a {
+    color: var(--primary-color);
+  }
+
+  a:hover {
+    color: var(--primary-color);
+  }
```
Comment (Keith-CY) [inline src/pages/Transaction/TransactionCell/styles.module.scss:309]: Unnecessary style

**Review by Keith-CY — APPROVED** [review]

**Comment by FrederLu** [comment]
image

https://ckb-explorer-frontend-in-magickbase-repo-6hw486sgy-magickbase.vercel.app/en/transaction/0x7d75674378935176fbf99ecdeffb37e7d9c3b3380eef73cdf3f6a4077d8775ca

image

1、 `Script info` button text is missing.

---
https://ckb-explorer-frontend-in-magickbase-repo-6hw486sgy-magickbase.vercel.app/en/address/tb1qd5q7hs0q0px6qj55j2nkkhjgm2zep7765zjyhs
image

image

2、The layout of the text on the pop-up window is abnormal when the browser displays it at 100%.

---
image

image

3、`Commitment check` and `Seal UTXO` information are missing under the RGB++ tab.

---
image

image

4、The mouse hover icon lacks prompt information.

**Comment by Daryl-L** [comment]
> image
> https://ckb-explorer-frontend-in-magickbase-repo-6hw486sgy-magickbase.vercel.app/en/transaction/0x7d75674378935176fbf99ecdeffb37e7d9c3b3380eef73cdf3f6a4077d8775ca
> 
> image
> 1、 `Script info` button text is missing.
> 
> https://ckb-explorer-frontend-in-magickbase-repo-6hw486sgy-magickbase.vercel.app/en/address/tb1qd5q7hs0q0px6qj55j2nkkhjgm2zep7765zjyhs image
> 
> image
> 2、The layout of the text on the pop-up window is abnormal when the browser displays it at 100%.
> 
> image image
> 3、`Commitment check` and `Seal UTXO` information are missing under the RGB++ tab.
> 
> image image
> 4、The mouse hover icon lacks prompt information.

The other 2 are removed by @Sven-TBD in Problem 3

**Comment by FrederLu** [comment]
UI draft
https://www.figma.com/design/6XNoimRDbFTTNm016rbIdU/Magickbase?node-id=37580-33772&t=W2kyAzw9MVM5wJRR-0

image
image
image

5、Cells that have been consumed have abnormal styles on the mobile terminal.

---
image
image

6、The icon style of the Live cell on the mobile terminal is abnormal.

---
image
image

7、The `BTC TX` and UI styles on the mobile terminal are inconsistent.

**Comment by Daryl-L** [comment]
> image
> https://ckb-explorer-frontend-in-magickbase-repo-6hw486sgy-magickbase.vercel.app/en/transaction/0x7d75674378935176fbf99ecdeffb37e7d9c3b3380eef73cdf3f6a4077d8775ca
> 
> image
> 1、 `Script info` button text is missing.
> 
> https://ckb-explorer-frontend-in-magickbase-repo-6hw486sgy-magickbase.vercel.app/en/address/tb1qd5q7hs0q0px6qj55j2nkkhjgm2zep7765zjyhs image
> 
> image
> 2、The layout of the text on the pop-up window is abnormal when the browser displays it at 100%.
> 
> image image
> 3、`Commitment check` and `Seal UTXO` information are missing under the RGB++ tab.
> 
> image image
> 4、The mouse hover icon lacks prompt information.

Solved except problem 4, it's not a bug. @Sven-TBD

**Comment by Keith-CY** [comment]
CI failed due to a typo at https://github.com/Magickbase/ckb-explorer-frontend/actions/runs/9644829646/job/26597743990?pr=345#step:3:41

**Comment by FrederLu** [comment]
https://ckb-explorer-frontend-in-magickbase-repo-6r10cb7v6-magickbase.vercel.app/transaction/0x789f0fe5adfd2fb41d9ecfbb7c2c9d03816c347da3c7a8733503a5a16867eb85
image

9、`BTC TX` can be displayed on a single line.

---

image

10、The size and position of `X` on mobile terminals still need to refer to UI adjustments.

**Review by FrederLu — APPROVED** [review]

**Pull request merged**